src/main/java/com/znsio/e2e/runner/Runner.java

```diff
@@ -88,27 +91,32 @@
     private static Map<String, Map> testDataForEnvironment;
     private static Map applitoolsConfiguration = new HashMap();
     private final Properties properties;
+    private final String DEFAULT_LOG_PROPERTIES_FILE = "src/main/resources/log4j.properties";
```

Comment:
Do we need this file name to be ./src/.....

Reply:
not required. Works without it!